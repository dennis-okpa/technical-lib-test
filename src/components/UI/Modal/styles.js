import styled from 'styled-components'

export const Overlay = styled.div`
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  padding: 5% 20%;
  cursor: pointer;
`

export const Img = styled.img`
  width: 50vw;
  max-height: 80%;
`
