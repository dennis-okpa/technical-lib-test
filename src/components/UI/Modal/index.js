import React from 'react'
import { Overlay, Img } from './styles'

function Modal({ img, handleClose }) {
  return (
    <Overlay onClick={handleClose}>
      <Img src={img} alt="large view" />
    </Overlay>
  )
}
export default Modal
