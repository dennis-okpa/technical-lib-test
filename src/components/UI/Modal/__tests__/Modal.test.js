import Modal from '../index'
import React from 'react'
import { render } from '@testing-library/react'

describe('Modal', () => {
  it('should render correctly', () => {
    const { container } = render(<Modal />)
    expect(container).toMatchSnapshot()
  })
})
