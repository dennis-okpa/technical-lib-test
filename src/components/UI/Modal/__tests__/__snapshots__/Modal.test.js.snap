// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal should render correctly 1`] = `
<div>
  <div
    class="sc-bdvvtL gxYpIB"
  >
    <img
      alt="large view"
      class="sc-gsDKAQ ixJPXD"
    />
  </div>
</div>
`;
